src/history.rs

```diff
@@ -253,6 +253,10 @@ fn umask() -> u16 {
 fn umask() -> libc::mode_t {
     unsafe { libc::umask(libc::S_IXUSR | libc::S_IRWXG | libc::S_IRWXO) }
 }
+#[cfg(not(any(windows, unix)))]
```

Comment:
Update line 248 with:
`#[cfg(any(windows, target_arch = "wasm32"))]`
And remove this block